src/node_worker.cc

```diff
@@ -459,8 +459,11 @@ Worker::~Worker() {
 
 void Worker::New(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
-  THROW_IF_INSUFFICIENT_PERMISSIONS(
-      env, permission::PermissionScope::kWorkerThreads, "");
+  CHECK(args[5]->IsBoolean());
+  if (args[5]->IsFalse()) {
```

Comment:
I would include a comment explaining why this is needed, or at least, explicitly saying args[5] refers to `isInternal`. Not a blocker though.